server/api/http/version.py

```diff
@@ -0,0 +1,14 @@
+from aiohttp import web
+
+async def get_version(request: web.Request):
+    try:
+        with open('VERSION', 'r') as version_file:
+            version_data = version_file.read()
+    except:
+        version_data = "0.0.0"
```

Comment:
I would just return a 500 error because if this happens, there is something wrong with the server.